.github/workflows/lint-and-test.yml

```diff
@@ -2,7 +2,8 @@ name: Lint and test
 
 on:
   push:
-    branches: [ $default-branch ]
+    branches-ignore:
```

Comment:
@vmarkovtsev if we run the CI for push and PR events it gets executed twice every time I push to a PR. Is this the optimal behaviour?
Do you know of any way to avoid launching the CI twice on pushes to PRs but still run it once after every push that is not part of a PR?